Libraries/Image/RCTShadowVirtualImage.m

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTShadowVirtualImage.h"
+#import "RCTImageLoader.h"
+#import "RCTImageUtils.h"
+#import "RCTBridge.h"
+#import "RCTConvert.h"
+#import "RCTUIManager.h"
+#import "RCTUtils.h"
+
+@implementation RCTShadowVirtualImage
```

Comment:
Not sure why it looks like I added the entire file - If you looks at the file it was added in October last year (91e6c98)